packages/foreman/rubygem-nokogiri/rubygem-nokogiri.spec

```diff
@@ -71,8 +71,8 @@ cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 mkdir -p %{buildroot}%{gem_extdir_mri}
-cp -a .%{gem_extdir_mri}/gem.build_complete %{buildroot}%{gem_extdir_mri}/
```

Comment:
This file is still needed since it signals Ruby that it should expect a .so file.

Reply:
I never knew that, thanks for the info!